packages/admin-panel/src/pages/resources/SurveyResponsesPage.js

```diff
@@ -36,10 +36,10 @@ const dateOfData = {
   Header: 'Date of Data',
   source: 'data_time',
   type: 'tooltip',
-  accessor: row => moment.parseZone(row.data_time).format('ddd, MMM Do YYYY, HH:mm:ss'),
+  accessor: row => moment.parseZone(row.data_time).toString(),
```

Comment:
This is the only change I don't get. Is the default `toString` the same as what we were using?